backend/app/common/exception/exception_handler.py

```diff
@@ -46,44 +48,48 @@ async def validation_exception_handler(request: Request, exc: RequestValidationE
         """
         message = ''
         data = {}
-        for raw_error in exc.raw_errors:
-            raw_exc = raw_error.exc
-            if isinstance(raw_exc, ValidationError):
-                if hasattr(raw_exc, 'model'):
-                    fields = raw_exc.model.__dict__.get('__fields__')
-                    for field_key in fields.keys():
-                        field_title = fields.get(field_key).field_info.title
-                        data[field_key] = field_title if field_title else field_key
-                # 处理特殊类型异常模板信息 -> backend/app/schemas/base.py: SCHEMA_ERROR_MSG_TEMPLATES
-                for sub_raw_error in raw_exc.raw_errors:
-                    sub_raw_exc = sub_raw_error.exc
-                    if isinstance(sub_raw_exc, (EnumMemberError, WrongConstantError)):
-                        if getattr(sub_raw_exc, 'code') == 'enum':
-                            sub_raw_exc.__dict__['permitted'] = ', '.join(
-                                repr(v.value)
-                                for v in sub_raw_exc.enum_values  # type: ignore
-                            )
-                        else:
-                            sub_raw_exc.__dict__['permitted'] = ', '.join(
-                                repr(v)
-                                for v in sub_raw_exc.permitted  # type: ignore
-                            )
-                # 处理异常信息
-                for error in raw_exc.errors()[:1]:
-                    field = str(error.get('loc')[-1])
-                    msg = error.get('msg')
-                    message += f'{data.get(field, field) if field != "__root__" else ""} {msg}' + '.'
-            elif isinstance(raw_error.exc, json.JSONDecodeError):
-                message += 'json解析失败'
+        raw_error = exc.raw_errors[0]
```

Comment:
The loop is no longer used because we end up returning only one error message, the loop outputs the last error message, and using slices outputs the first error message. 

This is done to be compatible with the logic in v2, making it less confusing during the migration.

Reply:
```
raw_error = exc.raw_errors[0] if exc.raw_errors else None
```
Would this way be better?

Reply:
In fact, if `raw_errors` is empty, `raw_errors[0]` will raise an error. The same error will occur for `for raw_error in exc.raw_errors`.

If add `if exc.raw_errors...`, an exception will be thrown only when `raw_exc = raw_error.exc` is encountered. Clearly, this is not compatible.